content/05_games/00_portal-2.cats

```diff
@@ -8,6 +8,7 @@ Name,Objective,OOB,SLA,"Pause Abuse","Demo Requirement",Leaderboard,Notes,Modera
 "Most Recent Community Chambers","Complete the specified amount of community maps from the Most Recent option.",No,No,No,N/A,https://www.speedrun.com/portal_2_category_extensions?h=Most_Recent_Community_Chambers-10-SP&x=zd363xrd-ylq9vjz8.21gjmx81-p85dqg0l.4qykr321,"Video proof is required. You can skip levels. You cannot play your own maps or maps you have previously seen.","Betsruner, BiSaXa, Fridge, phunkpai"
 "Co-op any% EHM","Complete up to the end of course 5 of the cooperative campaign.",Yes,Yes,No,200th,https://www.speedrun.com/portal_2_category_extensions?h=Co-op_Any_EHM&x=vdononok,"You may use glitches to skip courses and maps. EHM is allowed.","Betsruner, BiSaXa, Fridge, phunkpai"
-"Portal Placement Never Fail","Complete the specified campaign according to the standard ruleset for the underlying category.",Yes,-,-,"All runs",https://www.speedrun.com/portal_2_category_extensions?h=Portal_Placement_Never_Fail-SP&x=wkpo8182-38dw2v1n.8102832q,"You may use `sv_portal_placement_never_fail 1` (which requires `sv_cheats 1`). All other rules come from the underlying category (Inbounds No SLA, AMC, Co-op any%, or Solo Co-op).","Betsruner, BiSaXa, Fridge, phunkpai"
+"Portal Placement Never Fail","Complete the specified campaign according to the standard ruleset for the underlying category.",Yes,No,Yes,"All runs",https://www.speedrun.com/portal_2_category_extensions?h=Portal_Placement_Never_Fail-SP&x=wkpo8182-38dw2v1n.8102832q,"You may use `sv_portal_placement_never_fail 1` (which requires `sv_cheats 1`). All other rules come from the underlying category (Inbounds No SLA, AMC, Co-op any%, or Solo Co-op).","Betsruner, BiSaXa, Fridge, phunkpai"
 "Singleplayer 100 Portals","Complete the entire singleplayer campaign within 100 portals.",No,No,Yes,200th,https://www.speedrun.com/portal_2_category_extensions?h=Least_Portals_Challenge_Runs-SP_100_Portals&x=9d83ywl2-ylqqryzl.z1903g81,"You must place no more than 100 portals during the run. `sar_lphud` can be used to track portal shots.","Betsruner, BiSaXa, Fridge, phunkpai"
 "Co-op 69 Portals","Complete all courses (1-6) of the cooperative campaign within 69 portals.",Yes,Yes,No,200th,https://www.speedrun.com/portal_2_category_extensions?h=Least_Portals_Challenge_Runs-Co-op_69_Portals&x=9d83ywl2-ylqqryzl.814rjevq,"You must place no more than 69 portals during the run. `sar_lphud` can be used to track portal shots.","Betsruner, BiSaXa, Fridge, phunkpai"
+"No Major Glitches","Complete the entire singleplayer campaign.",No,No,No,5th,https://www.speedrun.com/portal_2_category_extensions?h=No_Major_Glitches&x=zdn458xk,"You must complete the game without the use of a major glitch, the list of major glitches can be found here: https://docs.google.com/document/d/10F62Cgars4I-kZ-alOjptZ64ozSMIJ4_j1r26kTGXHc/edit?usp=sharing","Betsruner, BiSaXa, Fridge, phunkpai"
```

Comment:
grammar: "...without the use of a major glitch. A list of major glitches can be found at https://..."